### ISSUE TITLE
Content that appears on hover or focus may be dismissed by the user 

### ISSUE THREAD

**Issue by Bluefish0558** [microsoft/vscode#227239]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.92.2
- OS Version: 14.6.1 (23G93)

Steps to Reproduce:

1. The setting is not specific to the current profile, and will retain its value when switching profiles" tooltip content that appears on hover, obscures other content, and cannot be dismissed without moving focus or the cursor.

2. Screen magnification users will have difficulty moving the cursor or focus off the trigger while still keeping the content behind the revealed content in view. Keyboard users will be unable to review the content behind the revealed content.

3. The best way to do this is to allow the hover/focus content to be hidden with the Esc key.

<img width="1389" alt="Hover on the content" src="https://github.com/user-attachments/assets/2259eacb-ea72-4b76-8993-9fd0b03cf70e">

**Comment by Yoyokrazy** [comment]
passing this to @meganrogge as I think this seems like a broader potential accessibility issue? Feel free to pass it forward to someone closer to the hover.

**Comment by meganrogge** [comment]
I'm not seeing that hover show up in insider's, are you @rzhao271 ?

**Comment by rzhao271** [comment]
That hover occurs when hovering over "Applies to all profiles"

I confirm that the Escape key currently does not hide the hover. Instead, when the user focuses the setting and shows the hover, pressing the escape key multiple times moves the focus back to the table of contents, then moves the focus back to the search input box, and then clears the search input box.

**Comment by rzhao271** [comment]
The issue is that keyboard focus is not on the hover, and pressing Escape in this case does not get picked up by the hover.
The hover has code to hide itself when pressing Escape, but that code only kicks in when the hover is focused.
I confirmed that this issue is true for hovers around the VS Code workbench in general, so I think this is potentially a broader accessibility issue @meganrogge.

The only ways to close the hover are to move focus elsewhere, to move the cursor elsewhere, or to specifically focus the hover and then press Escape. @Bluefish0558 already mentioned at least the first two approaches.

**Comment by meganrogge** [comment]
@rzhao271 does using ctrlCmd+k+I focus the hover? That's how workbench hovers work and are accessible.

**Comment by rzhao271** [comment]
@meganrogge `Ctrl+K I` shows a different hover instead in this case, but that's irrelevant to this issue. Will file a new issue.

Closing this issue because clicking on the hover and then pressing escape resolves the issue.
